compiler/native/compile.go

```diff
@@ -262,6 +263,7 @@ func (c *client) modifyConfig(build *yaml.Build, libraryBuild *library.Build, re
 		Pipeline: string(data),
 		Build:    libraryBuild.GetNumber(),
 		Repo:     repo.GetName(),
+		Org:      repo.GetOrg(),
```

Comment:
FWIW:

If you don't think it would break anything, we could just send the "full name" of the repo?

```suggestion
		Repo:     repo.GetFullName(),
```

Reply:
Wanted to leave the existing field intact to allow backwards compatibility with our internal implementation of the modification API.